.github/renovate.json5

```diff
@@ -0,0 +1,29 @@
+{
+  "extends": [
+    "config:best-practices",
+    "github>aquaproj/aqua-renovate-config#2.7.5"
+  ],
+  "schedule": [
+    "every month"
```

Comment:
Do we want to set this so it's on the 1st of the month so we get them as the same time as our other monthly PR waves?

Reply:
Sounds reasonable. Updated!